synthesizer/sed.py

```diff
@@ -43,17 +36,31 @@ class Sed:
     fnu = Quantity()  # nJy
```

Comment:
Missing `Quantity` for lam and lnu. Also comment is misleading. Units can be changed by the user. The global unit system is respected but doesn't mean it will always be nJy.